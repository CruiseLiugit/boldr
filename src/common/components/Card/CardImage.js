/* @flow*/
import React from 'react';

const CardImage = (props: { imgSrc: String }) => {
  return (
    <div className="boldr-card__image">
      <img src={ props.imgSrc } alt="card feature" className="boldr-card__img" />
    </div>
  );
};

export default CardImage;
