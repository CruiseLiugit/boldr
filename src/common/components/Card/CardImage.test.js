import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import CardImage from './CardImage';

it('<CardImage />, should render', () => {
  const wrapper = shallow(<CardImage />);
  const wrapperInner = wrapper.find('.boldr-card__image');
  expect(wrapperInner.is('.boldr-card__image')).toBe(true);
});

it('renders snapshot', () => {
  const wrapper = shallow(<CardImage />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
