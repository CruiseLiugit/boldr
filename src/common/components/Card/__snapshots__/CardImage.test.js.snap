exports[`test renders snapshot 1`] = `
<div
  className="boldr-card__image">
  <img
    alt="card feature"
    className="boldr-card__img" />
</div>
`;
